common/src/main/java/org/opensearch/ml/common/transport/upload_chunk/MLRegisterModelMetaInput.java

```diff
@@ -29,20 +33,26 @@ public class MLRegisterModelMetaInput implements ToXContentObject, Writeable{
 
     public static final String FUNCTION_NAME_FIELD = "function_name";
     public static final String MODEL_NAME_FIELD = "name"; //mandatory
-    public static final String DESCRIPTION_FIELD = "description";
+    public static final String DESCRIPTION_FIELD = "description"; //optional
+
+    public static final String VERSION_FIELD = "version";
     public static final String MODEL_FORMAT_FIELD = "model_format"; //mandatory
     public static final String MODEL_STATE_FIELD = "model_state";
     public static final String MODEL_CONTENT_SIZE_IN_BYTES_FIELD = "model_content_size_in_bytes";
     public static final String MODEL_CONTENT_HASH_VALUE_FIELD = "model_content_hash_value"; //mandatory
     public static final String MODEL_CONFIG_FIELD = "model_config"; //mandatory
     public static final String TOTAL_CHUNKS_FIELD = "total_chunks"; //mandatory
-    public static final String MODEL_GROUP_ID_FIELD = "model_group_id"; //mandatory
+    public static final String MODEL_GROUP_ID_FIELD = "model_group_id"; //optional
+    public static final String BACKEND_ROLES_FIELD = "backend_roles"; //optional
```

Comment:
I see all access control fields are optional now. Per my understanding, if user doesn't provide any of these, the model group will be private, right?